x_min: -2.25
x_max: 0.3
y_min: -1.05
y_max: 0.9
z_min: 0
z_max: 0.5